### ISSUE TITLE
no visible focus indicator for buttons

### ISSUE THREAD

**Issue by OutlawAndy** [RoleModel/optics#169]
**Describe the bug**
While buttons remain keyboard accessible, they appear not to be.  This behavior is not in compliance with the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/WCAG22/quickref/#focus-visible)

If this is intentional, please explain.

**Expected behavior**
some form of visual feedback that indicates focus.  An outline, for example.

**Screenshots**

no focus indicator
![2023-05-23 12 14 21](https://github.com/RoleModel/optics/assets/1504753/fab4f147-3b89-4af2-ba2d-a7d743c67c28)

default webkit focus indicator `outline: -webkit-focus-ring-color auto 1px;`
![2023-05-23 12 16 03](https://github.com/RoleModel/optics/assets/1504753/a8ee4e98-9602-4951-8a7a-14300e802f9b)

**Additional context**

We received a support request on Maverick, stating that the modal close button was inaccessible to students who must navigate via keyboard only.  I discovered that the modal didn't actually have a close `button`, just a `div` styled like one.  So my first instinct (being a genius after all) was to use a `button` instead of a `div`.  That lead me to notice that while the close button was now accessible, it still didn't appear to be.  Interestingly, leaving the `div` in place with the addition of `tabindex=0` provides both keyboard accessibility & a visual focus indicator.  However, that seems semantically incorrect and probably doesn't play very nicely with screen readers..

Full discloser: Maverick is not actually using Optics (I don't think, could be wrong) However, it's css base is the Rolemodel design system, which lead me to check into whether or not Optics had the same issue.  Thanks!

**Comment by OutlawAndy** [comment]
@Jeremy-Walton, @scriswell, @zoopmaster   - not a pressing issue, but let me know if any of you have an opinion on this.  Thanks!

**Comment by zoopmaster** [comment]
@OutlawAndy - Optics does have keyboard focus states. Is this an open Maverick support request that needs to be resolved?

**Comment by OutlawAndy** [comment]
@zoopmaster okay, thanks.  My mistake.  Closing this issue

**Comment by Jeremy-Walton** [comment]
@OutlawAndy I believe Maverick started in June of 2021 so while there is a lot that is similar, a lot about Optics has been improved since then.

Focus states on buttons have been in the last few version of Optics, but were revised slightly in the recent update. https://docs.optics.rolemodel.design/?path=/docs/components-button--docs

![Screenshot 2023-05-24 at 1 11 16 PM](https://github.com/RoleModel/optics/assets/5957102/69255cd5-1237-4749-91c8-dc281bf8d736)
